### ISSUE TITLE
TASK - Wireframes & Design

### ISSUE THREAD

**Issue by Daisy-McG** [Daisy-McG/Guess-The-Song#13]
Create wireframes and list out colour schemes

Colours should be contrast checked before decision is made to ensure correct contrast ratios as per WCAG
